test/Webhook.spec.js

```diff
@@ -48,6 +48,37 @@ describe('Webhooks', () => {
         expect(event.id).to.equal(EVENT_PAYLOAD.id);
       });
 
+      it('should return an Event instance from a payload and header with type Buffer', async () => {
+        const header = stripe.webhooks.generateTestHeaderString({
+          payload: EVENT_PAYLOAD_STRING,
+          secret: SECRET,
+        });
+
+        const event = await constructEventFn(
+          Buffer.from(EVENT_PAYLOAD_STRING),
+          Buffer.from(header),
+          SECRET
+        );
+
+        expect(event.id).to.equal(EVENT_PAYLOAD.id);
+      });
+
+      it('should return an Event instance from a payload and header with type Uint8Array', async () => {
+        const header = stripe.webhooks.generateTestHeaderString({
+          payload: EVENT_PAYLOAD_STRING,
+          secret: SECRET,
+        });
+
+        const textEncoder = new TextEncoder();
+        const event = await constructEventFn(
+          new Uint8Array(textEncoder.encode(EVENT_PAYLOAD_STRING)),
```

Comment:
textEncoder.encode returns a Uint8Array judging by the docs. Do we need extra construction?

Reply:
Oh good catch, we don't need the extra construction. Fixed!